pom.xml

```diff
@@ -85,6 +85,16 @@
             <version>1.7.12</version>
         </dependency>
         <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
```

Comment:
We had issues when both log4j and logback were in the class path when we were working on dynamic logger. Although that issue (I think) stemmed from us changing both logging frameworks log level programmatically. @sabertiger did you test this version with your service?